apps/challenges/models.py

```diff
@@ -177,6 +179,11 @@ def is_active(self):
     sender=Challenge,
     weak=False,
 )
+signals.post_save.connect(
```

Comment:
@KhalidRmb and @Ayukha Can you please coordinate and add this in only one PR.
Ref: #2814